ansible.cfg

```diff
@@ -1,6 +1,12 @@
 [ssh_connection]
 pipelining=True
-#ssh_args = -F ./ssh-bastion.conf -o ControlMaster=auto -o ControlPersist=30m
+#ssh_args=-o ForwardAgent=yes
+
```

Comment:
Let's get rid of empty lines. Just something shorter like:
```
#ssh_args=basic opts
# Uncomment the line below...
#ssh_args = -F ./ssh-bastion.conf...
```

Reply:
Makes sense. Will do.